hoomd/md/EvaluatorPairLJ.h

```diff
@@ -222,6 +222,17 @@ class EvaluatorPairLJ
             return false;
         }
 
+    DEVICE Scalar evalEnergyLRCIntegral()
+        {
+        // Note that lj1 and lj2 are defined above.
+        // lj1 = 4.0 * epsilon * pow(sigma,12.0)
+        // lj2 = 4.0 * epsilon * pow(sigma,6.0);
+        // Due to a bug, the way that lj1 and lj2 are defined will unfortunately be changing soon...
```

Comment:
Complete your PR first and then someone else will have to worry about it :)

It would be helpful if you included the analytical expression for the integral in the comment to help that developer.

Reply:
I think I've done what you've asked for--I added the equations in LaTeX in the comment.